yr/tutorials/Number_Guessing_Game.md

```diff
@@ -0,0 +1,512 @@
+
+Our final result should look like this:
+
+![Result](../images/number_guessing_game/image21.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+Let's summarize the logic flow of our <span class = "control"> conditional if/then/else if/then/else</span> block to better understand conditionals. First, we check whether the **if** statement is true: **myGuessSlot > secret**. If it is, then whatever is in the first **then** statement is run, and the other following statements are ignored. However, if the **if** statement is false, then we move on to checking whether the **else if** statement (**myGuessSlot < secret**) is true or not. If it is true, then the second **then** statement is run and the other following statements are ignored. If it is not true, then we move onto the **else** statement and run it.
+
+Our new guessing game host Skill is now finished! Let's send it to Amazon and test it out.
+
+# Test Your Skill! 
+
+## Send the Code to Amazon
+
+At the bottom of the testing box, click on **Send Updates**. 
+
+![Send updates](../images/number_guessing_game/image33.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+The updates should take about 1-2 minutes to finish updating. If you decide to make any changes to your Alexa Skill after completing this tutorial (e.g. updating/adding/deleting blocks), make sure you do this step again to send your new changes to Amazon! 
+
+## Test your Skill
+
+If you have an Alexa-enabled device handy, say *“Alexa, tell guessing game host to quiz me”* to it and hear the response. You can also do this on your mobile device by downloading the free **Amazon Alexa** app, which comes with a built-in Alexa simulator.
+
+If you don’t have an Alexa-enabled device on hand, the Testing page in App Inventor allows you to simulate an Alexa using your custom Alexa skill right in your browser! Simply type in the textbox at the bottom, *“tell guessing game host to quiz me”* (You can omit the Alexa at the beginning of the phrase when using the testing box). Does the conversation flow like we planned it to? Here’s an example conversation:
+
+Note that the "secret" number is randomly generated, so that the user will see different sequences than the example shown here.
+
+<p style="padding-bottom: 7px"></p>
+![Example Conversation](../images/number_guessing_game/image43.png){:.enlargeImage}
+<p style="padding-bottom: 7px"></p>
+
+## Finish!!
+
+Congratulations! You've finished programming the **Try Number tutorial** and learned about using **conditionals!**
+
+# Expand Your App
+
+Here are some ideas for ways to enhance your app! 
+
+The way our app is built as of now, once we guess the correct number, the game ends. Try changing the app so that you can "restart" the game once you guess the right number! 
+
+* Add a new line asking "Do you want to play again?" once the correct number is guessed.
```

Comment:
Do not tell them how to do this.  They may have other ways of implementing this than what you are suggesting here.